src/main/java/com/griefprevention/util/command/MonitoredCommands.java

```diff
@@ -207,17 +208,20 @@ private void addCommand(@NotNull Command command, @NotNull String suffix, @Nulla
         if (command instanceof BukkitCommand)
         {
             // If this is a command from Bukkit, it is in the same package.
-            if (BukkitCommand.class.getPackage().equals(command.getClass().getPackage()))
-                return "/bukkit:";
+            if (BukkitCommand.class.getPackage().equals(command.getClass().getPackage())) return "/bukkit:";
             // Otherwise this is probably a wrapper for a vanilla command.
-            else return "/minecraft:";
+            else
+                return "/minecraft:";
         }
 
-        // User-created commands.yml commands don't ever have a prefix, they're added directly to the map.
+        // User-created commands.yml commands don't ever have a prefix, they're added
+        // directly to the map.
         if (command instanceof FormattedCommandAlias) return null;
 
-        // There's no way for us to detect potential prefixes of commands that don't identify themselves.
-        // The only other way to tell is if the command happened to conflict and had its label reassigned.
+        // There's no way for us to detect potential prefixes of commands that don't
+        // identify themselves.
+        // The only other way to tell is if the command happened to conflict and had its
+        // label reassigned.
```

Comment:
This is the other type of comment mess that exists in a bunch of places. I don't think GP previously had a hard line wrap, so adding one in for comments in particular leaves a lot of trailing single words. Probably should have been a block comment (my bad), but here we are.

Reply:
Yea, no hard line wrap. Line wrap is enforced only on a subjective basis. In general, it is better to write stuff that doesn't need to be wrapped in the first place, otherwise the developer should choose a good place to wrap.